packages/venia-concept/src/components/Head/headProvider.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { HeadProvider as _HeadProvider } from 'react-head';
+
+const filterOutTitleTags = titleTags => {
+    titleTags
+        .filter(titleNode => !titleNode.dataset.synteticTag)
```

Comment:
synteticTag or syntheticTag?

Reply:
My bad. Spelling mistake.

Reply:
Can you drop a comment here to explain why we're removing some and not all `title` tags, and how `syntheticTag` plays into that?

Reply:
So the problem with `react-head` is, it won't render a `title` tag if there is one already present in the markup. So according to their docs, it is required to remove the existing tags and then render the needed ones using `react-head`. Now we should not be removing the tags that were created using `react-head` in case of a re-render. That is where the `synthetictag` flag comes in. If the `syntheticTag` flag is set, it means that title tag was created using `react-head` and should not be removed.

Reply:
> the problem with react-head is, it won't render a title tag if there is one already present in the markup.

Isn't that what [this test](https://github.com/magento/pwa-studio/pull/1537/files#diff-6ad3f434d87e531558aa3b745f33c3ecR40) does though? Or does that test only work because you added the synthetic tag handling?

Reply:
I meant drop a comment in the code itself so that in the future no one removes this line thinking that it's not needed 😛